### PR TITLE
GSoC: Week 7 Blogpost

### PR DESCRIPTION
**Pull request by JoaoDell**
Hello everyone, here is my week 7 blogpost:
image

### CODE REVIEW

File: docs/source/posts/2023/2023-07-17-week-7-joaodellagli.rst [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+
+After those results, I refactored my PR `#804 <https://github.com/fury-gl/fury/pull/804>`_ to better fit its current status, and it is 
+now ready for review. Success!
+
+
+This Week's Goals
+-----------------
+After finishing the first iteration of my experimental program, the next step is to work on an API for KDE rendering. I plan to meet 
+with my mentors and talk about the details of this API, so expect an update next week. Also, I plan to take a better look on my fellow GSoC FURY
+contributors work so when their PRs are ready for review, I will have be better prepared for it.
```
Comment (ganimtron-10) [inline docs/source/posts/2023/2023-07-17-week-7-joaodellagli.rst:63]: missing `to` in `I will have be better ...`

**Review by ganimtron-10 — COMMENTED** [review]
Hey @JoaoDell,
Nice Blogpost! I just had the below comment. PTAL.

**Review by tvcastillod — COMMENTED** [review]
Hi @JoaoDell, really nice and detailed blogpost. Everything looks good to me.

**Review by skoudoro — APPROVED** [review]
Great post, merging

**Pull request merged**